cli/command/context/completion.go

```diff
@@ -0,0 +1,46 @@
+// FIXME(thaJeztah): remove once we are a module; the go:build directive prevents go from downgrading language version to go1.16:
+//go:build go1.22
```

Comment:
Note to self; need to update https://github.com/docker/cli/pull/5868 after this is merged.